tests/index.js

```diff
@@ -52,6 +52,17 @@ describe('I18n', () => {
         expect(i18n.translate('new.item.b')).to.equal('BEE!');
     });
 
+    it('key exists without scope', () => {
+        expect(i18n.exists('root.user.name')).to.true;
```

Comment:
This is not the proper way to assert with chai.

What you're probably looking for is:
```js
expect(i18n.exists('root.user.name')).to.be.true;
```